src/redux/books/books.js

```diff
@@ -0,0 +1,34 @@
+const ADD_NEW_BOOK = 'bookstore/books/ADD_BOOK';
+const REMOVE_BOOK = 'bookstore/books/REMOVE_BOOK';
+
+export function addNewBook(newBook) {
+  return {
+    type: ADD_NEW_BOOK,
+    newBook,
+  };
+}
+
+export function removeBook(book) {
+  return {
+    type: REMOVE_BOOK,
+    book,
+  };
+}
+
+export default function reducer(state = [{
+  id: '1',
+  title: 'Test book one',
+  author: 'Test Author',
+}], action = {}) {
```

Comment:
- [ ] For your reducer, please remove the test data and set an initial state with an empty array 🙏
- [ ] Through the action, you get access to the type that was dispatched and also the payload. So action should not be set to an empty object. Please fix this 😎

Reply:
thanks, @codepantha, the action is being forced by eslint. 
it is for a default that will always be overridden by the values passed. it throws the error when I remove it. may be you can advise on another way to do it

Reply:
@isaacpitwa if that is the case, then go ahead. I honestly have never come across it being done that way, but if it's being enforced directly by eslint, then you're probably good to go 😎